spec/lib/hubspot/contact_list_spec.rb

```diff
@@ -178,27 +178,70 @@
     end
   end
 
-  describe '#add' do
-    cassette "add_contacts_to_lists"
+  let(:static_list) { Hubspot::ContactList.all(static: true, count: 3).last }
```

Comment:
Is `static_list` being used by other tests? It looks like in this change all instances of `static_list` have been removed.

Reply:
Good catch 🎣! Removed.